floodviz/static/js/floodviz_onready.js

```diff
@@ -17,12 +17,47 @@ document.addEventListener("DOMContentLoaded", function (event) {
 	FV.map_figure.init();
 
 	
-	// Hydrograph 
+	/* Hydrograph Setup */
+
+	// map interactions for hydrograph events
+	var maptip = d3.select('#maptip');
+	/**
+	 	* Shows/removes sitename tooltip on map figure at correct location.
+	 */
+	var show_map_tooltip = function(sitename, sitekey) {
+		var gage_point_cords = document.getElementById('map'+sitekey).getBoundingClientRect();
+		maptip.transition().duration(500);
+		maptip.style("display", "inline-block")
+			.style("left", (gage_point_cords.left) + 7 + "px")
+			.style("top", (gage_point_cords.top - 45) + "px")
+			.html((sitename));
+	}
```

Comment:
Can they be but in FV.mapmodule? Anything that changes the map visualization should be there.